dev-libs/mimalloc/mimalloc-2.0.5-r2.ebuild

```
@@ -0,0 +1,31 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib
+
+DESCRIPTION="mimalloc is a compact general purpose allocator with excellent performance"
```

In a separate commit, let's do this just while I've noticed:
```suggestion
DESCRIPTION="A compact general purpose allocator with excellent performance"
```

Also, while not obligated, feel free to join me on mimalloc. It's only (initially) in tree for mold.